backend/src/flotilla/settings/settings.env

```
@@ -1,5 +1,5 @@
 FLOTILLA_OPENAPI_CLIENT_ID=af1d4470-dbfd-41b3-835a-1900070ff0cb
-FLOTILLA_APP_CLIENT_ID=ea4c7b92-47b3-45fb-bd25-a8070f0c495c
+FLOTILLA_APP_CLIENT_ID=f5993820-b7e2-4791-886f-f9f5027dc7be
```

```suggestion
FLOTILLA_APP_CLIENT_ID=ea4c7b92-47b3-45fb-bd25-a8070f0c495c
```
We should only use the backend app reg, the frontend app reg should not exist